File: assignment 38/P1.java
import Mangesh.StringX;
import java.util.*;
class Show extends StringX
{
	int i=0;
	int j=0;
	int l=0;
	public void Dis()
	{
	int k=str.length();
	char arr[]=str.toCharArray();
	for(i=1;i<=k-1;i++)
	{
		for(j=1;j<=k;j++)
		{
			System.out.print(arr[j-1]+" ");
		}
		System.out.println("");
	}
    }
}
class P1
{
	public static void main(String[] args) 
	{
		Show sobj = new Show();
		sobj.Accept();
		sobj.Display();	
		sobj.Dis();
	}
}

File: assignment 21/p1.c
#include<stdio.h>
void Display()
{
int i=0;
for(i=0;i<=255;i++)
{
printf("%d\t%o\t%x\t%c \n",i,i,i,i);
}
}
int main()
{
Display();
return 0;
}